docs/source/03_tutorial/01_spaceflights_tutorial.md

```diff
@@ -6,7 +6,7 @@
 
 In this tutorial, we illustrate the typical Kedro workflow and the steps necessary to convert an empty Kedro project template into a working project.
 
-In the text, we assume that you create a empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately 2 hours and you will learn each step of the Kedro project development workflow, by working on an example to construct nodes and pipelines for the price-prediction model.
+In the text, we assume that you create an empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately 2 hours and you will learn each step of the Kedro project development workflow, by working on an example to construct nodes and pipelines for the price-prediction model.
```

Comment:
Might make sense to address this on the same line:

```suggestion
In the text, we assume that you create an empty project and follow the flow of the tutorial by copying and pasting the example code into the project as we describe. This tutorial will take approximately 2 hours, and you will learn each step of the Kedro project development workflow by working on an example to construct nodes and pipelines for the price-prediction model.
```

Both "This tutorial will take approximately 2 hours" and "you will learn each step of the Kedro project development workflow" are independent clauses; however "working on an example to construct nodes and pipelines for the price-prediction model" is not and the comma splice should be removed.